docs/guides/configuration-guide/loadbalancer.md

```diff
@@ -83,16 +125,44 @@ is known everywhere and the self-signed certificates are accepted as valid.
    ```
 
 3. Use in OpenStack
+   * Add the custom CA to the configuration repository in the directory `environments/kolla/certificates/ca` with the same
+     name like in step 1
+   * Configure the custom CA to be copied to the OpenStack containers
+     ```yaml title="environments/manager/configuration.yml"
+     kolla_copy_ca_into_containers: "yes"
+     openstack_cacert: /etc/ssl/certs/ca-certificates.crt
+     ```
+
+4. Import the ca certificate to all nodes so that the custom CA is known everywhere and the self-signed certificates are accepted as valid.
+   ```
+   osism apply certificates
+   ```
+
+5. Execute all steps in the [general procedure](#general-procedure) above
+
+### Generating TLS certificates with Let’s Encrypt
 
-   The custom CA must also be copied into the OpenStack containers. To do this, the custom
-   CA is first added in a file in the `environments/kolla/certificates/ca` of the configuration
-   repository.  It makes sense to use the same filename like in step 1.
+Using Let's encrypt certificates is a good alternative to traditional certificate authorities and
+greatly simplifies the administration of TLS certificates.
 
-   The import of the custom CA must then be explicitly enabled.
+For a working Let's Encrypt configuration, the API endpoints (configured by `kolla_internal_fqdn` and `kolla_external_fqdn`) 
+must be accessible from the internet.
```

Comment:
This is not necessarily true anymore since you can configure distinct ACME servers for internal or external usage, they only need to be reachable from the control nodes, see:

https://docs.openstack.org/kolla-ansible/latest/admin/tls.html#generating-tls-certificates-with-let-s-encrypt

so you can now use a private ACME-CA-Server in an air gapped environment.